diff --git a/tidy-html5/Makefile b/tidy-html5/Makefile
index 07ce58b..ec1c4c7 100644
--- a/tidy-html5/Makefile
+++ b/tidy-html5/Makefile
@@ -18,6 +18,8 @@ PKG_HASH:=59c86d5b2e452f63c5cdb29c866a12a4c55b1741d7025cf2f3ce0cde99b0660e
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=README/LICENSE.md README/LICENSE.txt
 
+CMAKE_INSTALL:=1
+
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
 
@@ -37,12 +39,24 @@ define Package/tidy-html5/conffiles
 /opt/etc/tidy.conf
 endef
 
+define Package/libtidy
+	SECTION:=libs
+	CATEGORY:=Libraries
+	TITLE:=HTACG HTML Tidy
+	URL:=http://www.html-tidy.org/
+	MAINTAINER:=Entware team, https://entware.net
+endef
+
 CMAKE_OPTIONS += \
-	-DBUILD_SHARED_LIB=OFF \
 	-DSUPPORT_LOCALIZATIONS=OFF \
 	-DTIDY_CONFIG_FILE="/opt/etc/tidy.conf" \
 	-DXSLTPROC_FOUND=OFF
 
+define Package/libtidy/install
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/libtidy.so* $(1)/opt/lib
+endef
+
 define Package/tidy-html5/install
 	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/tidy $(1)/opt/bin
@@ -50,4 +64,5 @@ define Package/tidy-html5/install
 	$(INSTALL_CONF) ./files/tidy.conf $(1)/opt/etc
 endef
 
+$(eval $(call BuildPackage,libtidy))
 $(eval $(call BuildPackage,tidy-html5))
