--- a/src/github.com/AdguardTeam/AdGuardHome/internal/aghnet/net_linux.go
+++ b/src/github.com/AdguardTeam/AdGuardHome/internal/aghnet/net_linux.go
@@ -30,7 +30,7 @@ func ifaceHasStaticIP(ifaceName string)
 		filePath string
 	}{{
 		checker:  dhcpcdStaticConfig,
-		filePath: "/etc/dhcpcd.conf",
+		filePath: "/opt/etc/dhcpcd.conf",
 	}, {
 		checker:  ifacesStaticConfig,
 		filePath: "/etc/network/interfaces",
@@ -139,13 +139,13 @@ func ifaceSetStaticIP(ifaceName string)
 	gatewayIP := GatewayIP(ifaceName)
 	add := updateStaticIPdhcpcdConf(ifaceName, ipNet.String(), gatewayIP, ipNet.IP)
 
-	body, err := ioutil.ReadFile("/etc/dhcpcd.conf")
+	body, err := ioutil.ReadFile("/opt/etc/dhcpcd.conf")
 	if err != nil {
 		return err
 	}
 
 	body = append(body, []byte(add)...)
-	err = maybe.WriteFile("/etc/dhcpcd.conf", body, 0o644)
+	err = maybe.WriteFile("/opt/etc/dhcpcd.conf", body, 0o644)
 	if err != nil {
 		return fmt.Errorf("writing conf: %w", err)
 	}
