--- a/src/github.com/AdguardTeam/AdGuardHome/internal/sysutil/net_linux.go
+++ b/src/github.com/AdguardTeam/AdGuardHome/internal/sysutil/net_linux.go
@@ -31,10 +31,10 @@ func ifaceHasStaticIP(ifaceName string)
 		filePath string
 	}{{
 		checker:  dhcpcdStaticConfig,
-		filePath: "/etc/dhcpcd.conf",
+		filePath: "/opt/etc/dhcpcd.conf",
 	}, {
 		checker:  ifacesStaticConfig,
-		filePath: "/etc/network/interfaces",
+		filePath: "/opt/etc/network/interfaces",
 	}} {
 		var f *os.File
 		f, err = os.Open(check.filePath)
@@ -140,13 +140,13 @@ func ifaceSetStaticIP(ifaceName string)
 	gatewayIP := GatewayIP(ifaceName)
 	add := updateStaticIPdhcpcdConf(ifaceName, ipNet.String(), gatewayIP, ipNet.IP)
 
-	body, err := ioutil.ReadFile("/etc/dhcpcd.conf")
+	body, err := ioutil.ReadFile("/opt/etc/dhcpcd.conf")
 	if err != nil {
 		return err
 	}
 
 	body = append(body, []byte(add)...)
-	err = file.SafeWrite("/etc/dhcpcd.conf", body)
+	err = file.SafeWrite("/opt/etc/dhcpcd.conf", body)
 	if err != nil {
 		return err
 	}
